KolibriShortcut/KolibriShortcut.vcxproj

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>16.0</VCProjectVersion>
+    <Keyword>Win32Proj</Keyword>
+    <ProjectGuid>{5A0416CD-8F36-4F72-9A42-A9FF5F0823B6}</ProjectGuid>
+    <RootNamespace>KolibriShortcut</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <ProjectName>Endless Key Launcher</ProjectName>
```

Comment:
Is that the user visible name of the app?

Reply:
Yes, that's the final binary name, you can see the exe file generated in the artifact here: https://github.com/endlessm/windows-usb-launcher/suites/3187982814/artifacts/73564029

Reply:
Perfect, thanks.